language/stdlib/modules/libra_system.mvir

```diff
@@ -46,22 +57,43 @@ module LibraSystem {
       // Only callable by the Association address
       assert(get_txn_sender() == 0xA550C18, 1);
 
-      move_to_sender<BlockMetadata>(BlockMetadata { height: 0 });
+      // TODO: How should we get the default block metadata? Should it be set in the first block prologue transaction or
```

Comment:
Good question! I think the answer is probably in the genesis because otherwise, the first non-genesis block cannot begin with a block metadata transaction.

Reply:
The default value is for the genesis. i.e: What should those values be in the genesis block?